csharp/src/Ice/SlicStream.cs

```diff
@@ -105,13 +96,11 @@ protected override async ValueTask<bool> ReceiveAsync(ArraySegment<byte> buffer,
         protected override async ValueTask ResetAsync(long errorCode) =>
```

Comment:
unrelated to this PR, why we don't use `ulong` here as we do for `StreamResetBody`

Reply:
Good question... the varulong can fit in a C# long value so there's no concern with overflows. Then I thought it would be less surprising to use long... that's also what the C# Quic API uses. That said, this error code isn't really used anyway. I guess it could eventually be used with other language mappings if we provide a way to provide to the Cancel method.